### PR TITLE
Update Facilitator Fee documentation to include information to suppor…

### PR DESCRIPTION
**Pull request by ShreyaThapa**
…t VCR bank to balance fee collection

### CODE REVIEW

File: pages/changelog.mdx [hunk 1]
```diff
@@ -21,11 +21,17 @@ Keep up to date on new product features and enhancements, API docs, tools change
 
 ## Completed
 
+### 2022-10-31
+
+**ADDED**
+
+- Added support for including [facilitator fee](/concepts/facilitator-fee) when creating transfers from Verified Customer's Bank into their Dwolla balance.
```
Comment (spencerhunter) [inline pages/changelog.mdx]: smol change to pluralize. Change facilitator fee to `facilitator fees`

File: pages/changelog.mdx [hunk 2]
```diff
@@ -21,11 +21,17 @@ Keep up to date on new product features and enhancements, API docs, tools change
...
+
 ### 2022-10-27
 
 **ADDED**
 
-- Added the `correlationId` attribute to the [webhook payload](/concepts/webhook-events#webhook-payload) for transfer related webhooks if a value was specified on transfer creation.
+- Added the `correlationId` attribute to the [webhook payload](/concepts/webhook-events#webhook-payload) for transfer related webhooks if a value was specified on transfer creation. Previously, fees could only be applied to transactions between two parties.
```
Comment (spencerhunter) [inline pages/changelog.mdx]: Should `Previously, fees could only be applied to transactions between two parties.` be included in the new changelog entry?

**Review by spencerhunter — CHANGES_REQUESTED** [review]

File: pages/changelog.mdx (hunk 2 above)
Comment (ShreyaThapa) [reply pages/changelog.mdx]: lol I added it to the wrong spot! Thanks for catching that! xD

File: pages/changelog.mdx (hunk 1 above)
Comment (ShreyaThapa) [reply pages/changelog.mdx]: Updated [here](https://github.com/Dwolla/dev-portal/pull/439/commits/08d8a1f9dfa6c889f3e976111cac931891b39478).

File: pages/changelog.mdx (hunk 2 above)
Comment (ShreyaThapa) [reply pages/changelog.mdx]: Corrected [here](https://github.com/Dwolla/dev-portal/pull/439/commits/08d8a1f9dfa6c889f3e976111cac931891b39478).

**Review by spencerhunter — APPROVED** [review]

**Pull request merged**